thesis_tex/titlepage.tex

```diff
@@ -29,10 +29,11 @@
 \end{spacing}
 
 \begin{tabular}{ll}
-\large Author:     & \large \author     \\[2mm]
-\large Supervisor: & \large \supervisor \\[2mm]				
-\large Advisor:	   & \large \advisor    \\[2mm]
-\large Date:       & \large \date
+\large Author:           & \large \author     \\[2mm]
+\large Supervisor:       & \large \supervisor \\[2mm]				
+\large Advisor:	         & \large \advisor    \\[2mm]
+\large Start Date:       & \large \startdate  \\[2mm]
```

Comment:
Start Date may not appear in the final thesis!
https://www.in.tum.de/in/fuer-studierende/pruefungen-und-formalitaeten/abschlussarbeit/